elasticsearch_collectd.py

```diff
@@ -668,6 +670,7 @@ def configure_callback(conf):
     log.info('host: %s' % c.es_host)
     log.info('port: %s' % c.es_port)
     log.info('es_index: %s' % c.es_index)
+    log.info('es_index_separate_graphx: %s' % c.es_index_separate_graph)
```

Comment:
graph instead of graphx?

Reply:
Yes, this is typo. I'll try on monday to send fix.